samples/attested_tls/non_enc_client/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.11)
+
+set(CMAKE_CXX_STANDARD 11)
+
+add_executable(tls_non_enc_client client.cpp verify_callback.cpp verify_signer_openssl.cpp)
+
+target_include_directories(tls_non_enc_client PRIVATE
+  ${CMAKE_CURRENT_SOURCE_DIR}
+  ${CMAKE_CURRENT_BINARY_DIR}
+  -I/usr/include/openssl)
```

Comment:
If we need this and `ssl` then we probably want to be using https://cmake.org/cmake/help/latest/module/FindOpenSSL.html

Reply:
Good idea. 
Replaced ssl with OpenSSL::SSL after adding find_package(OpenSSL REQUIRED)